cmd/thanos/store.go

```diff
@@ -495,6 +495,25 @@ func runStore(
 			s.Shutdown(err)
 		})
 	}
+
+	// Start bloom name filter updater.
+	{
+		ctx, cancel := context.WithCancel(context.Background())
+		level.Debug(logger).Log("msg", "setting up periodic update for label names")
+		g.Add(func() error {
+			return runutil.Repeat(10*time.Second, ctx.Done(), func() error {
+				level.Debug(logger).Log("msg", "Starting label names update")
```

Comment:
nit (also in other places):

```suggestion
				level.Debug(logger).Log("msg", "starting label names update")
```